qiskit/providers/ibmq/api_v2/rest/schemas/job.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Schemas for job."""
+from marshmallow.validate import OneOf
+from qiskit.providers.ibmq.apiconstants import ApiJobStatus
+from qiskit.validation import BaseSchema
+from qiskit.validation.fields import String, Url, Nested, Number, Boolean
+
+
+# Helper schemas.
+
+class InfoQueueResponseSchema(BaseSchema):
+    """Nested schema for StatusResponseSchema"""
+
+    # Required properties
+    status = String(required=True)
+    position = Number(required=True, missing=0)
+
+
+class JobResponseSchema(BaseSchema):
+    """Nested schema for CallbackUploadResponseSchema"""
+    # pylint: disable=invalid-name
+
+    # Required properties
+    id = String(required=True)
+    kind = String(required=True)
+
+
+# Endpoint schemas.
+
+class PropertiesResponseSchema(BaseSchema):
+    """Schema for PropertiesResponse"""
+    pass
+
+
+class StatusResponseSchema(BaseSchema):
+    """Schema for StatusResponse"""
+
+    # Optional properties
+    infoQueue = Nested(InfoQueueResponseSchema, required=False)
+
+    # Required properties
+    status = String(required=True, validate=OneOf([status.name for status in ApiJobStatus]))
+
+
+class CancelResponseSchema(BaseSchema):
+    """Schema for CancelResponse"""
+
+    # Optional properties
+    error = String(required=False)
+
+    # Required properties
+    cancelled = Boolean(required=True, description="True if job can be cancelled, else False. "
```

Comment:
I don't think we specifically look for this property. Yes it's weird to have just `error` as the only one. Perhaps we can reintroduce this after confirming with api *and* update the mainline code.

Reply:
We inexplicably do! 

Under `qiskit.providers.ibmq.job.ibmqjob:288` which checks for `cancelled` field in a response of the form `{'cancelled': True}`.

Reply:
Are we looking at the same code? To me that [block](https://github.com/Qiskit/qiskit-ibmq-provider/blob/master/qiskit/providers/ibmq/job/ibmqjob.py#L288) has 
```
        try:
            response = self._api.cancel_job(self._job_id)
            self._cancelled = 'error' not in response
```

Reply:
Yep, I misread sorry. I was reading it as `'cancelled' in response`, I'll remove this field from the schema. Thank you! :)